tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -469,4 +469,49 @@ describe('People And Roles component for Change of Registration', () => {
     expect(wrapper.vm.$data.activeIndex).toBe(NaN)
     expect(wrapper.vm.$data.isAddingEditingOrgPerson).toBe(true)
   })
+
+  it('change button is not visible to users for SP where the sole proprietor is a non-individual', () => {
```

Comment:
You can be specific here: a non-individual must be an organization.

Reply:
Good call, I'll change it.